### ISSUE TITLE
Add person pages with a curation summary

### ISSUE THREAD

**Issue by kimrutherford** [pombase/website#332]
From https://github.com/pombase/website/issues/328#issuecomment-300785820

>Also, just a thought, could the name link to a person page with perhaps a curation summary for
> the person (number of annotations added for number of publications)
>
> even further into the future..
> People could be able to claim pages (like on the biosharing resource) to add info like contact details,
> affiliations, or whatever (and link a pombase account with canto account....then you can have the
> option of snazzy customization of gene pages etc etc)

kmr: and links to https://orcid.org/ once we have the ORC IDs in Canto/Chado.

**Comment by kimrutherford** [comment]
> kmr: and links to https://orcid.org/ once we have the ORC IDs in Canto/Chado.

Now I've thought about it, I think we should only have person pages for users with ORCIDs.  I think that's the best way to uniquely identify users.

**Comment by mah11** [comment]
Eventually we could also offer a web form for people to opt in to having a person page, inspired by and similar to what SGD and WormBase do (we can make ORCID a mandatory field).

at SGD:
http://www.yeastgenome.org/cgi-bin/colleague/colleagueSearch
if none of the matches are you, you can enter a new record - results link to http://www.yeastgenome.org/cgi-bin/colleague/colleagueUpdate

WormBase is a little more opaque, much like the rest of their interface, but it starts here:
http://www.wormbase.org/resources/person
it's a lot less obvious how to add a completely new Person record there, so let's not copy them

**Comment by ValWood** [comment]
This is essentially what Jurg's group asked about @Antonialock  add anything  to this ticket

**Comment by Antonialock** [comment]
Essentially we thought that it could act as an incentive for community curation if the name specified in the community curation "attribution" linked to a persons page showing the number of papers curated by said person and the number of annotations they contributed.

**Comment by ValWood** [comment]
yep... that would work and is along the lines we were thinking for people to use in their grant "data dissemination" sections too...

**Comment by ValWood** [comment]
from https://github.com/pombase/website/issues/591
via @Antonialock 

Jurg seemed a bit upset his stats won't look so good if we make "people pages" #332  - unless we add a details for # of sessions delegated :-) 

It might be nice to also have "lab" pages. 
The lab page could list publications associated with the lab, and stats (number of papers community curated, annotations made...? etc) and a link to the lab webpage (entered at point of community curation?)

A lab name could be created based on the name of the last author of publications (e.g. Bahler lab, Wu lab, Gould lab...).

I think this should work for most publications. I can think of two cases where it is not ideal:
- two labs might collaborate. I guess more often than not however this would involve one pombe lab and one technique/biological area-specialized lab, who probably wouldn't care about pombe stats pages anyway, so not a biggy
- new lab heads might still use an old supervisor as last author to get into more high profile journals (probably not many cases...but not ideal)

**Comment by ValWood** [comment]
So we could use the old relegated "corresponding author" for this. Many are actually still filled in. We could default them, and then give a quick check over.

We def need the option to add >1 person to this field. Some are  multi lab....

**Comment by kimrutherford** [comment]
I think this issue has some overlap with the APICURON project.

**Comment by ValWood** [comment]
Yes,  if you spot any tickets related to attribution, ORIDs etc tag with APICURON. We can discuss on Monday Morning.
